examples/advanced/example.js

```diff
@@ -15,6 +15,7 @@ import {
   type EditMode,
   ModifyMode,
   TranslateMode,
+  TransformMode,
```

Comment:
Can you also add this mode to the documentation at `docs/api-reference/modes/overview.md`?